caffe2/core/nomnigraph/Converters/Caffe2.cc

```diff
@@ -13,44 +13,54 @@ getArgumentsFromOperator(caffe2::OperatorDef op) {
   return argMap;
 }
 
+nom::repr::NeuralNetOperator::NNLayout getLayout(std::map<std::string, caffe2::Argument> argMap) {
+  if (argMap.count("order")) {
```

Comment:
Its better to use `argMap.find` and compare result to `argMap.end()`. In current implementation you do the lookup twice (first to check if an element exists, then to retrieve it).